src/shaders/symbol_sdf.fragment.glsl

```diff
@@ -15,21 +17,24 @@ in vec3 v_data1;
 #pragma mapbox: define lowp float halo_width
 #pragma mapbox: define lowp float halo_blur
 
-void main() {
-    #pragma mapbox: initialize highp vec4 fill_color
-    #pragma mapbox: initialize highp vec4 halo_color
-    #pragma mapbox: initialize lowp float opacity
-    #pragma mapbox: initialize lowp float halo_width
-    #pragma mapbox: initialize lowp float halo_blur
+float median(vec3 values) {
+    return max(min(values.r, values.g), min(max(values.r, values.g), values.b));
+}
 
-    float EDGE_GAMMA = 0.105 / u_device_pixel_ratio;
+float screenPxRange(float fontScale) {
+    // WebGL1 doesn't support derivatives and we can't use version 300
```

Comment:
I think this comment is a little confusing because it doesn't say _why_ we'd want to support derivatives? i.e. what the preferred code would be and why it would be better